src/specter/views/hwi.py

```diff
@@ -18,22 +18,7 @@
 """
     Support for calling the 'hwi' CLI. See note below in _enumerate()
 """
-HWI_DIR = None
-def _locate_hwi():
-    # Is hwi globally available?
-    returned_output = subprocess.check_output(["which", "hwi"])
-    if not returned_output:
-        # Are we in a virtualenv?
-        virtualenv_bin_dir = sys.executable[:sys.executable.rfind(os.sep)]
-        returned_output = subprocess.check_output(["hwi", "--version"], cwd=virtualenv_bin_dir)
-        if "hwi" not in returned_output.decode("utf-8").lower():
-            print("Could not find 'hwi' executable for HWI hardware wallet support")
-            exit(0)
-        else:
-            HWI_DIR = virtualenv_bin_dir
-
-_locate_hwi()
-
+HWI_EXEC = which("hwi")
```

Comment:
@kdmukai i did some simplifications here. Hope that works for you as well.

Reply:
Cool. My original workaround was so hack-y and I hated that it was necessary. I don't have the cycles to help with testing at the moment but definitely Concept ACK.